Sources/SWBCSupport/CLibclang.h

```diff
@@ -126,6 +126,8 @@ bool libclang_has_cas_up_to_date_checks_feature(libclang_t lib);
 /// Whether the libclang has current working directory optimization support.
 bool libclang_has_current_working_directory_optimization(libclang_t lib);
 
+bool libclang_has_reproducer_feature(libclang_t lib);
```

Comment:
Please rebase and tag this with CSUPPORT_EXPORT, and do the same for libclang_scanner_generate_reproducer so it gets the __declspec(dllexport) modifier on windows